microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -887,6 +890,8 @@ The history methods allow back and forward navigation in WebView2, and the histo
 
 The `NavigationStarting` event allows the app to cancel navigating to specified URLs in WebView2, including for frames.
 
+<!-- TODO: basic links such as NavigationStarting should not be only in this "Block unwanted navigating" section; they should primarily be in section "Navigation events", which ought to be moved to above this "Block unwanted navigating" section -->
```

Comment:
@vbryh-msft, @nishitha-burman - 

The "Navigation events" minor section 
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=win32cpp#navigation-events
should be the first section within the major section "Navigate to pages and manage loaded content".
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=win32cpp#navigate-to-pages-and-manage-loaded-content

The "Navigation events" section should contain `NavigationStarting` links (which are now only in the "Block unwanted navigating"
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=win32cpp#block-unwanted-navigating
section).

Reply:
In the Nav Events article in this PR, there's now a table that lists the events in order and that links to the various sections of the "Overview of APIs" article.  

The "Overview of APIs" article doesn't have a single section that lists all the nav events in order, but instead they are scattered across about 5 sections.  

The "Block unwanted navigating" section is above the more general "Navigation events" section that lists several nav events.

Reply:
keeping as-is per offline discussion